pulp_container/tests/functional/api/test_crud_distributions.py

```diff
@@ -1,297 +1,49 @@
-        self._test_spaces()
-        self._test_begin_slash()
-        self._test_end_slash()
-
-    def _test_spaces(self):
-        """Test that spaces can not be part of ``base_path``."""
-        self._try_create_distribution(base_path=utils.uuid4().replace("-", " "))
-        self._try_update_distribution(
-            self.distribution.pulp_href, base_path=utils.uuid4().replace("-", " ")
-        )
-
-    def _test_begin_slash(self):
-        """Test that slash cannot be in the begin of ``base_path``."""
-        self._try_create_distribution(base_path="/" + utils.uuid4())
-        self._try_update_distribution(self.distribution.pulp_href, base_path="/" + utils.uuid4())
-
-    def _test_end_slash(self):
-        """Test that slash cannot be in the end of ``base_path``."""
-        self._try_create_distribution(base_path=utils.uuid4() + "/")
-        self._try_update_distribution(self.distribution.pulp_href, base_path=utils.uuid4() + "/")
-
-    def _test_unique_base_path(self):
-        """Test that ``base_path`` can not be duplicated."""
-        body = gen_distribution()
-        body["base_path"] = body["base_path"].replace("-", "/")
-        distribution_data = ContainerContainerDistribution(**body)
-        distribution_response = self.distribution_api.create(distribution_data)
-        created_resources = monitor_task(distribution_response.task).created_resources
-
-        self.distribution = self.distribution_api.read(created_resources[0])
-        self.addCleanup(self.namespace_api.delete, self.distribution.namespace)
-        self._try_create_distribution(base_path=self.distribution.base_path)
-
-    def test_valid_base_paths(self):
-        """Test the validation for base paths."""
-        distributions_data = [
-            ContainerContainerDistribution(**gen_distribution(base_path="0000001/000/000")),
-            ContainerContainerDistribution(**gen_distribution(base_path="0000002/a_a/000")),
-            ContainerContainerDistribution(**gen_distribution(base_path="0000003/a_a/0-0")),
-            ContainerContainerDistribution(**gen_distribution(base_path="0000004/a__a/000")),
-            ContainerContainerDistribution(**gen_distribution(base_path="0000005/a__a/a.a")),
-        ]
-        for data in distributions_data:
-            task = monitor_task(self.distribution_api.create(data).task)
-            distribution = self.distribution_api.read(task.created_resources[0])
-            self.addCleanup(self.namespace_api.delete, distribution.namespace)
-
-    def test_invalid_base_paths(self):
-        """Test the validation for base paths."""
-        self._try_create_distribution(base_path="i/am/an/very:bad:repo/name.py")
-        self._try_create_distribution(base_path="invalid_")
-        self._try_create_distribution(base_path="_invalid")
-        self._try_create_distribution(base_path="another/invalid..reponame")
-        self._try_create_distribution(base_path="also/invalid___reponame")
```

Comment:
I think these tests (or maybe the whole class) got lost. Were they redundant?

Reply:
Yeah, we already test these in pulpcore afaik.